0005-configuration-management-extension/README.md

```diff
@@ -142,19 +230,47 @@ Example:
 }
 ```
 
-A delivery confirmation response must be sent to the next topic:
-```
- <endpoint_token>/push/json/status
+#### Configuration push response
+
+The client MUST respond to the configuration push request by publishing the response message to the `/status` resource path according to the [request/response design pattern defined in 1/KP](/0001-kaa-protocol/README.md#requestresponse-pattern).
```

Comment:
Why not respond with 404 Not found? (Which perfectly fits the bill because the resource path points to an absent endpoint_token)